reconcile/utils/jenkins_api.py

```diff
@@ -8,20 +8,28 @@
 from reconcile.utils.secret_reader import SecretReader
 
 
+def init_jenkins_from_secret(
```

Comment:
Would prefer this to be a class method of the JenkinsApi class.

Reply:
Done